almanac.lic

```diff
@@ -14,6 +14,9 @@ class Almanac
     @almanac_skills = settings.almanac_skills
     @almanac = settings.almanac_noun
 
+    @startup_delay = settings.almanac_startup_delay || 15
```

Comment:
This does not need to be an instance var (`@`)

Reply:
Fixed. Thanks again for your reviews!